main.go

```diff
@@ -140,6 +151,57 @@ func setupRouter() *gin.Engine {
 		c.JSON(http.StatusOK, result.Update.State == "capturing")
 	})
 
+	r.GET("/calendar", func(c *gin.Context) {
+		client := &http.Client{}
+		// Cutoff is set to 24 hours from now
+		cutoff := time.Now().UnixMilli() + int64(86400000)
```

Comment:
Do we actually need the explicit conversion to an int64 here?

Reply:
Nope, we actually don't :D